src/frontend/hooks/server/invites.ts

```diff
@@ -1,88 +1,17 @@
 import {useClientApi} from '@comapeo/core-react';
-import {
-  useMutation,
-  useQueryClient,
-  useSuspenseQuery,
-} from '@tanstack/react-query';
+import {useSuspenseQuery} from '@tanstack/react-query';
+const ROOT_QUERY_KEY = '@comapeo/core-react';
 
-import {useActiveProject} from '../../contexts/ActiveProjectContext';
-import {useActiveProjectIdActions} from '../../contexts/ActiveProjectIdStoreContext';
-import {ALL_PROJECTS_KEY, PROJECT_MEMBERS_KEY} from './projects';
-
-export const INVITE_KEY = 'pending_invites';
+function getPendingInvitesQueryKey() {
+  return [ROOT_QUERY_KEY, 'invites', {status: 'pending'}] as const;
+}
```

Comment:
would add a comment that mentions that this is copied from [here](https://github.com/digidem/comapeo-core-react/blob/v3.2.0/src/lib/react-query/invites.ts#L23-L25)